src/main/scala/uk/gov/hmrc/releaser/Releaser.scala

```diff
@@ -198,7 +198,7 @@ object Releaser {
     val artefactBuilder = ArtefactMetaData.fromFile _
 
     val coordinator = new Coordinator(stageDir, artefactBuilder, verifyGithubCommit, gitHubTagAndRelease)
-    val repoFinder = new Repositories(metaDataGetter)(Seq(mavenRepository, ivyRepository)).findReposOfArtefact _
+    val repoFinder = new Repositories(metaDataGetter)(Seq(mavenRepository, ivyRepository, gradleRepository)).findReposOfArtefact _
```

Comment:
Since both mavenRepository and gradleRepository are looking for the BintrayMetadata of a given artifact in the same repository (i.e release-candidates) which ever is first on this line will win and hence will get a wrong RepoFlavour, in this case always MavenRepo and hense the desired effect in the file prefix will not be achieved.